OrbitBase/ThreadUtilsTest.cpp

```diff
@@ -51,3 +52,43 @@ TEST(ThreadUtils, GetSetThreadNames) {
   EXPECT_NE(pthread_setname_np(pthread_self(), kLongThreadName.data()), 0);
 #endif
 }
+
+TEST(ThreadUtils, GetThreadName) {
+  absl::Mutex mutex;
+#if _WIN32
+  using PidType = uint32_t;
+#else
+  using PidType = pid_t;
+#endif
+  PidType other_tid = 0;
+  bool other_name_read = false;
+  static const std::string kThreadName = "OtherThread";
```

Comment:
add a test for empty name?

Reply:
Added for the current thread. Also rearranged the tests a bit.